src/languageServerInstaller.ts

```diff
@@ -40,21 +39,15 @@ export class LanguageServerInstaller {
 			throw err;
 		}
 
-		// Do not wait on the showInformationMessage
-		vscode.window.showInformationMessage(`Installed terraform-ls ${currentRelease.version}.`, "View Changelog")
-			.then(selected => {
-				if (selected === "View Changelog") {
-					return vscode.env.openExternal(vscode.Uri.parse(`https://github.com/hashicorp/terraform-ls/releases/tag/v${currentRelease.version}`));
-				}
-			})
+		this.showChangelog(currentRelease.version);
 	}
 
 	async installPkg(release: Release, installDir: string, userAgent: string): Promise<void> {
 		const destination = `${installDir}/terraform-ls_v${release.version}.zip`;
 		fs.mkdirSync(installDir, { recursive: true }); // create install directory if missing
 	
-		let os = goOs();
-		let arch = goArch();
+		const os = platformOs();
+		const arch = platformArch();
```

Comment:
the _go_ in the name here was actually somewhat intentional, because the names can be Go specific - hence the conditions translating `win32` to `windows` or `x64` to `amd64`.

Reply:
Ah, that makes sense. I was thinking of it as a language server thing rather than the Go environment.